DailyQuickstart/ViewController.swift

```diff
@@ -0,0 +1,157 @@
+
+        // Disable the idle timer, so the screen will remain active while the app is in use.
+        UIApplication.shared.isIdleTimerDisabled = true
+
+        // Handle UI updates.
+        self.updateControls()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        // Automatically join the room specified by `roomURLString` when this view controller appears.
+        self.enterRoom()
+    }
+
+    // Updates the UI according to the inputs (e.g camera off/on).
+    private func updateControls() {
+        // Set the image for the camera button.
+        cameraInputButton.setImage(
+            UIImage(systemName: callClient.inputs.camera.isEnabled ? "video.fill": "video.slash.fill"),
+            for: .normal
+        )
+
+        // Set the image for the mic button.
+        microphoneInputButton.setImage(
+            UIImage(systemName: callClient.inputs.microphone.isEnabled ? "mic.fill": "mic.slash.fill"),
+            for: .normal
+        )
+    }
+
+    private func enterRoom() {
+        guard let roomURL = URL(string: roomURLString) else {
+            return
+        }
+
+        // This call is where you would add a meeting token to join a private room.
+        callClient.join(url: roomURL, token: nil) { result in
+            print(result)
+
+            // You can either handle the join event in a callback or in a delegate method.
+        }
+    }
+
+    // MARK: - Button Actions
+
+    @IBAction func didTapToggleCamera(_ sender: Any) {
+        // Disable or enable the camera.
+        let isEnabled = self.callClient.inputs.camera.isEnabled
+        self.callClient.setInputEnabled(.camera, !isEnabled)
+    }
+
+    @IBAction func didTapToggleMicrophone(_ sender: Any) {
+        // Disable or Enable the microphone.
+        let isEnabled = self.callClient.inputs.microphone.isEnabled
+        self.callClient.setInputEnabled(.microphone, !isEnabled)
+    }
+
+    @IBAction func didTapLeaveRoom(_ sender: Any) {
+        self.callClient.leave()
+    }
+}
+
+// MARK: - CallClientDelegate
+
+extension ViewController: CallClientDelegate {
+    // Handles a remote participant joining.
+    func callClient(_ callClient: CallClient, participantJoined participant: Participant) {
+        print("Participant \(participant.id) joined the call.")
+
+        // Create a new view for this participant's video track.
+        let videoView = VideoView()
+        
+        // Determine whether the video input is from the camera or screen.
+        let cameraTrack = participant.media?.camera.track
+        let screenTrack = participant.media?.screenVideo.track
+        let videoTrack = screenTrack ?? cameraTrack
+        
+        // Set the track for this participant's video view.
+        videoView.track = videoTrack
+        
+        // Add this participant's video view to the dictionary.
+        self.videoViews[participant.id] = videoView
```

Comment:
We don't get one of these calls for the local participant, so there's no way to add a view for them here.